### PR TITLE
Add dashboard API plugins 

### PR DESCRIPTION
**Pull request by Mt-is-at-O**
## List of necessary new plugins
- [x] RobotMode
- [x] PayLoad
- [x] Do
- [x] Arch
- [x] CP
- [x] SetCollisionLevel
- [x] GetAngle
- [x] GetPose
- [x] DI
- ModbusCreate
- ModbusClose
- GetInBits
- GetInRegs
- GetCoils
- SetCoils
- GetHoldRegs
- SetHoldRegs
- RunScript
- StopScript
- PauseScript
- ContinueScript

### CODE REVIEW

File: mg400_interface/src/commander/dashboard_commander.cpp
Comment (Mt-is-at-O) [inline mg400_interface/src/commander/dashboard_commander.cpp:1]: I modified `mg400_interface` to use Arch plugin.
- fix arch function
- delete unnecessary `arch.hpp` file

File: mg400_msgs/srv/DI.srv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+uint32 index
```
Comment (m12watanabe1a) [inline mg400_msgs/srv/DI.srv]: INDEX constant can be here?

File: mg400_node/include/mg400_node/mg400_node.hpp [hunk 2]
```diff
@@ -31,13 +31,28 @@ class MG400Node : public rclcpp::Node
 {
 private:
   const std::vector<std::string> default_dashboard_api_plugins_ = {
+    "mg400_plugin::AccJ",
```
Comment (m12watanabe1a) [inline mg400_node/include/mg400_node/mg400_node.hpp]: I want to remove those from default plugins, those are optionals.

**Review by m12watanabe1a — APPROVED** [review]
LGTM

**Pull request merged**